report/sections/titlepage.tex

```diff
@@ -0,0 +1,22 @@
+\begin{titlepage}
+  
+  \begin{center}
+    \begin{LARGE}
+      Department of Electrical and Computer Engineering \\
+      University of Victoria \\
+      ELEC 483 - Digital Video Processing \\[1cm]
+      \textsc{Invisible watermarks}
```

Comment:
Are you thinking of co-op reports? Journal articles use two columns and have the abstract in a column ([example](http://paginas.fe.up.pt/~jca/wrsc/templates/IEEE-Conference-A4-format.pdf))

As an aside, I have a variant of this in a two-column format. Gonna do a side-by-side when this is done~